x-pack/legacy/plugins/siem/public/pages/network/index.tsx

```diff
@@ -6,18 +6,39 @@
 
 import React from 'react';
 import { Redirect, Route, Switch } from 'react-router-dom';
+
 import { pure } from 'recompose';
 
+import { i18n } from '@kbn/i18n';
 import { NetworkComponentProps } from '../../components/link_to/redirect_to_network';
 
 import { IPDetails } from './ip_details';
 import { Network } from './network';
+import { PageRoute } from '../../components/page_router/pageroute';
 
 export const NetworkContainer = pure<NetworkComponentProps>(({ match }) => (
   <>
     <Switch>
-      <Route strict exact path={match.url} component={Network} />
-      <Route path={`${match.url}/ip/:ip`} component={IPDetails} />
+      <Route
+        strict
+        exact
+        path={match.url}
+        render={props => (
+          <PageRoute
+            {...props}
+            component={Network}
+            title={i18n.translate('xpack.siem.pages.Network.NetworkTitle', {
```

Comment:
For the most part our translation keys have been lower case separated by dots:

Examples:
```ts
xpack.siem.hosts.kqlPlaceholder
xpack.siem.navigation.overview
```

Should these just be `xpack.siem.pages.network.networkTitle` with camel case?

Reply:
👍